### ISSUE TITLE
Body:setLinearVelocity(x,y) is broken

### ISSUE THREAD

**Issue by slime73** [love2d/love#391]
**[Original report](https://bitbucket.org/rude/love/issue/386) by Anonymous.**

----------------------------------------

I'm trying to compensate delay when networking using LUBE, however I got stuck for quite a few time trying to make my code work, when I tried using Body:setLinearVelocity(0,1). It worked as intended, no big surprise, so then I tried Body:setLinearVelocity(1,0). Then I tried Body:setLinearVelocity(1,1) and it worked once more, but then I tried Body:setLinearVelocity(vX,vY) and It didn't work! I even printed vX and vY right before using setLinearVelocity and they show valid values, and if I hardcode those values in Body:setLinearVelocity() it works too!
TLDR: setLinearVelocity(x,y) doesn't work if BOTH parameters are variables.
This is Love2D 0.7.2 for Windows. Haven't tried it in any other OS.
